app/scripts/controllers/registration.js

```diff
@@ -73,7 +73,11 @@ angular.module('confRegistrationWebApp')
       });
     }, 15000);
 
-    $scope.goToNext = function () {
+    $scope.goToNext = function (registrationPageForm) {
+      if(!registrationPageForm.$valid && (registrationPageForm.$error.max || registrationPageForm.$error.min)){
```

Comment:
@OzzieOrca  This basically checks if the value entered by the registrant for questions with min / max range set, is within the range. If it is not, the page is not submitted and a validation error message is displayed in the page.